src/app/shared/components/template/components/toggle-bar/toggle-bar.ts

```diff
@@ -39,7 +39,7 @@ export class TmplToggleBarComponent
 
   constructor(private platform: Platform) {
     super();
-    this.mode = this.platform.is("android") ? "md" : "ios" || "ios";
+    this.mode = this.platform.is("ios") ? "md" : "android" || "android";
```

Comment:
I don't understand what this code is doing, we don't have ios deploy targets so that part is unecessary and mixing ternary (`? ... :`) with logical or (`||`) is a bit of an anti-pattern (both are trying to do the same thing of handling if-else scenarios. If it can't be made to work with one then just use an `if` statement for clarity

Reply:
https://stackoverflow.com/questions/42026158/precedence-logical-or-vs-ternary-operator